Cargo.toml

```diff
@@ -38,6 +39,7 @@ default = [
 account_balance = ["dep:openssl"]
 b2b = ["dep:openssl"]
 b2c = ["dep:openssl"]
+bill_manager = []
```

Comment:
Do you think we should we tie the `chrono` dependency to this feature? 🤔

Reply:
Yeah, makes sense to do so.